src/@next/Typography/Typography.tsx

```diff
@@ -1,4 +1,9 @@
 import React, { ElementType, ReactNode } from 'react';
+import '@fontsource/noto-sans/400.css';
```

Comment:
importing on `index.ts` for some reason didn't work...
importing here actually applies this to all of the component in storybook